lib/phoenix_live_view/html_engine.ex

```diff
@@ -1309,10 +1309,10 @@ defmodule Phoenix.LiveView.HTMLEngine do
          _attr,
          id?
        ) do
-    if value in ~w(ignore append prepend replace) do
+    if value in ~w(ignore stream append prepend replace) do
       validate_phx_attrs!(t, meta, state, "phx-update", id?)
     else
-      message = "the value of the attribute \"phx-update\" must be: ignore, append or prepend"
+      message = "the value of the attribute \"phx-update\" must be: ignore, stream, append or prepend"
```

Comment:
```suggestion
      message = "the value of the attribute \"phx-update\" must be: ignore, stream, append, prepend or replace"
```

Maybe that was missed in this commit? https://github.com/phoenixframework/phoenix_live_view/commit/5e1149c6cfde2fd7cbba2862dffc56f20398202e